src/three-components/SmoothControls.ts

```diff
@@ -42,6 +42,8 @@ export interface SmoothControlsOptions {
   // A scalar in 0..1 that changes the dampening factor, corresponding to
   // factors from 0.05..0.3
   dampeningScale?: number;
+  // a percentage representing how much the model can be rotated beyond max min angles
```

Comment:
Is this really a percentage? We generally use ratios [0,1], instead of percentages [0,100].